pkg/asset/ignition/machine/master_ignition_customizations.go

```diff
@@ -45,10 +46,12 @@ func (a *MasterIgnitionCustomizations) Generate(dependencies asset.Parents) erro
 	defaultPointerIgnition := pointerIgnitionConfig(installConfig.Config, rootCA.Cert(), "master")
 	savedPointerIgnition := master.Config
 
-	if savedPointerIgnition != defaultPointerIgnition {
+	savedPointerIgnitionJson, err := ignition.Marshal(savedPointerIgnition)
+	defaultPointerIgnitionJson, err := ignition.Marshal(defaultPointerIgnition)
```

Comment:
since any error is ignored anyway,
```suggestion
	savedPointerIgnitionJson, _ := ignition.Marshal(savedPointerIgnition)
	defaultPointerIgnitionJson, _ := ignition.Marshal(defaultPointerIgnition)
```

Reply:
Ah thanks, I actually meant to add error handling but since the config just got unmarshal'd in the config asset I guess this should be fine.